pkg/stis/calstis/cs7/scalewcs.c

```diff
@@ -69,6 +70,8 @@ CoordInfo *coord_o  io: size and coordinate info for output
 	    plate_scale = coord_o->cdelt[1];	/* CDELT2 */
 	else if (sts->dispaxis == 2)
 	    plate_scale = coord_o->cdelt[0];	/* CDELT1 */
+	else
+	    assert(0);
```

Comment:
../pkg/stis/calstis/cs7/scalewcs.c:70:11: warning: variable 'plate_scale' is used uninitialized whenever 'if' condition is false [-Wsometimes-uninitialized]
        else if (sts->dispaxis == 2)
                 ^~~~~~~~~~~~~~~~~~
[247/615] Compiling pkg/stis/calstis/cs11/gettimes11.c
../pkg/stis/calstis/cs7/scalewcs.c:74:24: note: uninitialized use occurs here
        sts->plate_scale[0] = plate_scale / sts->ltm[0];
                              ^~~~~~~~~~~
../pkg/stis/calstis/cs7/scalewcs.c:70:7: note: remove the 'if' if its condition is always true
        else if (sts->dispaxis == 2)
             ^~~~~~~~~~~~~~~~~~~~~~~
../pkg/stis/calstis/cs7/scalewcs.c:40:20: note: initialize the variable 'plate_scale' to silence this warning
        double plate_scale;     /* before scaling according to binning */
                          ^
                           = 0.0